gap-analysis/index.md

```diff
@@ -49,22 +49,23 @@ The platform APIs in general have not provided much support for mathematical not
 * $x^2$ -- "x squared" versus "x superscript 2 end superscript"
 * $\hat{x}$ -- "x hat" versus "x modified above with circumflex"
 * $\big(\begin{smallmatrix} 1 & 0\\\\ 0 & 1\end{smallmatrix}\big)$ -- "the 2 by 2 identity matrix" vs "open paren start 2 by 2 table;  row 1, column 1, 1, column 2, 0, row 2, column 1 0, column 2 1, end table, close paren"
 
 While all AT speaks $x^2$ semantically, some do not speak $\hat{x}$ semantically, and none currently recognize an identity matrix.
 
-Some notations such as fractions and square roots sometimes require bracketing words or tones to indicate the start and end of some notations for someone who can't see the notation. For example, $\sqrt{x+1}$ is unambiguously spoken as "the square root of x plus 1 end root". Without these bracketing words, "the square root of x plus 1" could also be interpreted as $\sqrt{x}+1$. However, for someone who is dyslexic and uses AT, the extra words are a distraction and shouldn't be used. This difference between the needs of users means that any solution that is adopted should allow flexibility based on the reader -- simple text strings generated by authors may be in appropriate for some readers. 
+Higher mathematics adds an additional tier of complexty. In it, even $x^2$ can be ambiguous, and may warrant a different reading than ordinary. One such case is $L^2$ read simply "L2" in the domain of Lebesgue spaces. In such texts, the author can often find themselves to be one of a only a handful of practitioners who have full grasp of their technical terminology, making manual annotation a frequent need for good AT.
+
+Some notations, such as fractions and square roots, may require bracketing words or tones to indicate the start and end of an expression, for readers who can't see the presentation. For example, $\sqrt{x+1}$ is unambiguously spoken as "the square root of x plus 1 end root". Without these bracketing words, "the square root of x plus 1" could also be interpreted as $\sqrt{x}+1$. However, for someone who is dyslexic and uses AT, the extra words are a distraction and shouldn't be used. This difference between the needs of users means that a real solution should allow flexibility based on the reader -- literal text strings annotated by authors may be inappropriate for some readers.
```

Comment:
> ...means that a real solution...

"real" sounds weird -- as opposed to a "fake" solution. I suggest "good" instead

Reply:
Maybe "full" (solution)? "production-grade"  doesn't seem appropriate. It is true for research projects, prototypes, etc.

Reply:
"full" is a great fit actually, done!